### PR TITLE
Get exercise list

### PR DESCRIPTION
**Pull request by JungAn2**
I don't know why but I did both issue #10 and #11.

### CODE REVIEW

**Review by lamont1008 — COMMENTED** [review]
looks good

**Review by Desss64 — APPROVED** [review]
Reviewed changes and everything looks good

File: prisma/migrations/20220509165823_ab/migration.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- CreateTable
+CREATE TABLE "preference" (
```
Comment (wrathofrathma) [inline prisma/migrations/20220509165823_ab/migration.sql:2]: Delete the migration files

File: app/services/ExercisesService.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+   * Fetch number of exercise on page
+   *
+   * @param {string} page page string from params
+   * @param {number} max maximumm exercise per page
+   * @return {string []} List of exercise in max number
+   * @throws {NotFoundException} Throws not found exception when exceeding max number
+   */
+  fetchPage(page: string, pagination: number): string[] {
+    try {
+      const max: number = Number(pagination);
```
Comment (wrathofrathma) [inline app/services/ExercisesService.ts:16]: why are we typecasting this

File: app/services/ExercisesService.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    return JSON.parse(JSON.stringify({ exercises: obj }));
+  },
+
+  /**
+   * Gets all the file names in exercise folder
+   *
+   * @return {string[]} List of exercises in the folder
+   * @throws {NotFoundException} Throws error when folder does not exist
+   */
+  getFileName(): string[] {
```
Comment (wrathofrathma) [inline app/services/ExercisesService.ts:88]: Change the name of this function to be more descriptive. 

I thought it was fetching the file name for a specific file and my brain went boom

File: app/services/ExercisesService.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  /**
+   * Parses the object into JSON
+   * with in object exercises
+   *
+   * @param {string []} obj object to make into JSON
+   * @return {JSON} returns object in JSON in form of {exercises: obj}
+   */
+  parseToJSON(obj: string[]): JSON {
+    return JSON.parse(JSON.stringify({ exercises: obj }));
+  },
```
Comment (wrathofrathma) [inline app/services/ExercisesService.ts:80]: 👀

**Review by wrathofrathma — APPROVED** [review]

**Pull request merged**